src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
- [ ] please make sure that you implement the application by following the requirements 
- [ ] ![image](https://user-images.githubusercontent.com/61790587/173924370-b50466b5-26e8-43a0-bd9e-f34a1d2af560.png)

Reply:
@youmari Hi Youmari, I think I understood it the wrong way, could you clarify with me, please? Is that markup of the Recent Scores supposed to be created dynamically with JS, right?